modules/cells/src/main/java/dmg/cells/nucleus/SystemCell.java

```diff
@@ -356,15 +356,15 @@ public void uncaughtException(Thread t, Throwable e)
         Throwable root = Throwables.getRootCause(e);
         if (root instanceof FileNotFoundException) {
             if (root.getMessage().contains("Too many open files")) {
-                _log.error(AlarmMarkerFactory.getMarker(PredefinedAlarm.OUT_OF_FILE_DESCRIPTORS,
+                logger.error(AlarmMarkerFactory.getMarker(PredefinedAlarm.OUT_OF_FILE_DESCRIPTORS,
                                                         getCellDomainName(),
                                                         getCellName()),
-                           "Uncaught exception in thread " + t.getName(),
+                           "Uncaught exception in thread {}", t.getName(),
```

Comment:
this one is an unfortunate change, as you change from log.error(String s, Throwable t) to  log.error(String format, Object o1, Object o1). The first method will log the stack trace as wee, while the second option will show only message from the exception.